sage-ios/SAGE/SAGE/ProfileCheckinViewController.swift

```diff
@@ -29,6 +29,7 @@ class ProfileCheckinViewController: UITableViewController {
         NSNotificationCenter.defaultCenter().addObserver(self, selector: "verifiedCheckinAdded:", name: NotificationConstants.addVerifiedCheckinKey, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: "unverifiedCheckinAdded:", name: NotificationConstants.addUnverifiedCheckinKey, object: nil)
         self.user = user
+        self.setNoContentMessage("You have no checkins. Go volunteer!")
```

Comment:
if a person is viewing checkins from a past semester, they cant volunteer for that past semester lel